apps/codeforafrica/src/lib/api.ghost.js

```diff
@@ -0,0 +1,88 @@
+import GhostContentAPI from "@tryghost/content-api";
+
+import convertToCamelCase from "@/codeforafrica/utils/camelcaseKeys";
+
+const { GHOST_URL } = process.env;
+// const { GHOST_URL } = "https://longform.codeforafrica.org";
+const { GHOST_API_KEY } = process.env;
+
+export async function getAllPosts(options) {
+  const { limit = 10, page = 1, ...other } = options || {};
+
+  const api = new GhostContentAPI({
+    url: GHOST_URL,
+    key: GHOST_API_KEY,
+    version: "v3.0",
+  });
```

Comment:
Move this into a function initializeAPI so that you don't have to duplicate the code.